packages/access-api/src/utils/ucan-log.js

```diff
@@ -0,0 +1,105 @@
+import pRetry from 'p-retry'
+import * as API from '../bindings.js'
+
+/**
+ *
+ * @param {object} input
+ * @param {URL} input.url
+ * @param {string} [input.auth]
+ */
+export const connect = (input) => new UCANLog(input)
+
+export const debug = () => new UCANLogDebug()
+
+/**
+ * @implements {API.UCANLog}
+ */
+class UCANLog {
+  /**
+   * @param {object} input
+   * @param {URL} input.url
+   * @param {string} [input.auth]
+   */
+  constructor({ url, auth }) {
+    this.url = url
+    this.auth = auth
+  }
+
+  /**
+   * @param {Uint8Array} car
+   */
+  async logInvocations(car) {
+    await pRetry(
+      () =>
+        fetch(this.url, {
+          method: 'POST',
+          headers: {
+            Authorization: `Basic ${this.auth}`,
+            'Content-Type': 'application/invocations+car',
```

Comment:
Is this defined anywhere?
nit: If it's ad-hoc, `application/vnd.`... might be appropriate.